src/VirtualClient/VirtualClient.Actions/Sysbench/SysbenchConfiguration.cs

```diff
@@ -52,7 +52,14 @@ protected override async Task ExecuteAsync(EventContext telemetryContext, Cancel
                 {
                     if (!cancellationToken.IsCancellationRequested)
                     {
-                        await this.PrepareMySQLDatabase(telemetryContext, cancellationToken);
+                        if (this.Benchmark == BenchmarkName.OLTP)
+                        {
+                            await this.PrepareOLTPMySQLDatabase(telemetryContext, cancellationToken);
+                        }
+                        else if (this.Benchmark == BenchmarkName.TPCC)
+                        {
+                            await this.PrepareTPCCMySQLDatabase(telemetryContext, cancellationToken);
+                        }
```

Comment:
else 
{
   throw new DependencyException("The '{this.Benchmark}' benchmark is not supported with the Sysbench workload.", ErrorReason.NotSupported);
}